Bottom: ddfb232cd8662ca03dc1fe9224976120a33be45f
Top:    3ef61f8fb4c34bad16524fd56bd5a0541c917bb8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 00:40:53 +0000

Refresh of hid-gtk-disable-gl-alpha-blend

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index ace9851..be8f39e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1085,8 +1085,8 @@ ghid_pinout_preview_expose (GtkWidget *widget,
 
 #if 0  /* We disable alpha blending here, as hid_expose_callback() does not
         * call set_layer() as appropriate for us to sub-composite rendering
-        * from each layer. If we leave alpha-blending on, it means text and
-        * overlapping pads are rendered looking very ugly.
+        * from each layer when drawing a single element. If we leave alpha-
+        * blending on, it means text and overlapping pads are rendered ugly.
         */
 
   glEnable (GL_BLEND);
